ccmbioinfo/lfs
# CARDS-1441: Upgrade material ui and mui/lab to version 5+

**Testing instructions**
* **All functionality** in **all projects** on **all screen sizes** needs to be tested
* Due to mui v5 changes, gray color for buttons is no longer supported, and the default color is primary (dark blue). We replaced the gray look with outline variant. Check that button color/variant is appropriate across the app (all Close/Cancel buttons should be outlined).
  * Note that some buttons have intentionally changed color (e.g. "Healthcare provider" on the DATA-PRO landing page, "Decline" in the DATA-PRO Terms of use dialog)

**Known issues**
* The properties listing appears too condensed in the questionnaire wizard - not a blocker, can be adjusted as part of 1767.
